### PR TITLE
Upgrade jacoco plugin

### PR DESCRIPTION
**Pull request by ajaygeorge**
Upgrade jacoco plugin

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,4 @@
 language: java
 
 jdk:
-  - oraclejdk8
-  - oraclejdk9
-  - openjdk10
-  - openjdk11
-  - openjdk-ea
+  - openjdk8
```
Comment (caithagoras) [inline .travis.yml:4]: Do we know what is the impact of this change?

**Review by caithagoras — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (ajaygeorge) [reply .travis.yml:4]: builds were failing with oracle jdks for a long time.
Updated https://github.com/prestodb/airbase/pull/12 to add in open jdk 10 and 11 as well.
will be removing the change from this PR.

**Comment by ajaygeorge** [comment]
@caithagoras removed the `.travis.yml` . If the changes look good can you please merge this PR.

**Comment by ajaygeorge** [comment]
Thank you @caithagoras !

**Pull request merged**